docs/content/preview/explore/ysql-language-features/advanced-features/partitions.md

```diff
@@ -99,7 +99,40 @@ To create a new partition that contains only the rows that don't match the speci
 CREATE TABLE order_changes_default PARTITION OF order_changes DEFAULT;
 ```
 
-Optionally, you can create an index on the key columns and other indexes for every partition, as follows:
+Optionally, you can create indexes on a partitioned table by executing the following command:
```

Comment:
I'm sure this was a suggestion from an earlier review, but "as follows" is shorter. You can even omit it, too, if you like.

```suggestion
Optionally, you can create indexes on a partitioned table as follows:

```